rules.md

```diff
@@ -52,45 +52,40 @@ nav_order: 2
 # ***§4*** Do not be a nuisance to other players
 **4.1** Spamming any type of chats with repeated words, repeated messages, pinging users or any other unwanted behaviour that could be considered disruptive to chatting is not allowed.
 
-**4.3** Inviting players to other servers, sharing server links, etc is not allowed, unless explicitly allowed by a member of staff.
+**4.2** Inviting players to other servers, sharing server links, etc is not allowed, unless explicitly allowed by a member of staff.
 
-**4.4** Insulting other players in any way, especially if repeated is not allowed.
+**4.3** Insulting other players in any way, especially if repeated is not allowed.
 
-**4.5** Using claims to block other players is not allowed.
+**4.4** Using claims to block other players is not allowed.
 
-**4.6** Squatting claims with no interest of making use of them, including for resale, is not allowed.
+**4.5** Squatting claims with no interest of making use of them, including for resale, is not allowed.
 
-**4.7** Coercing players into installing software that is potentially malicious or dangerous is not allowed.
 
-**4.8** Minor use of profanity is okay, but excessive use of profanity is not allowed.
+**4.6** Use of profanity is okay, but disruptive use of profanity is not allowed.
 
-**4.9** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
+**4.7** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay, but should be undertaken with caution.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.7.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay.
 
-**4.10** Explicit content of any form, including but not limited to: adult themes, erotic imagery, explicit innuendo, self harm and suicide, gore, shock content, etc. is not allowed.
+**4.8** Explicit content including: self harm and suicide, gore, shock content, etc. is not allowed.
```

Comment:
NSFW (erotic content) should not be allowed.

Reply:
Let's not forget there are underaged people playing the server, it is unethical to allow this.

Reply:
Whoa there, its not that I had a different position, Im not about to allow people to post nsfw content on channels, now discussing stuff reasonably like adults is something different

Reply:
Reverted